omnibus/config/projects/chef.rb

```diff
@@ -51,15 +51,27 @@
 # Chef Release version pinning
 override :chef, version: "local_source"
 override :ohai, version: "master"
+override :"rb-readline", version: "v0.5.3"
+
+# Global FIPS override flag.
+if windows? || rhel?
+  override :fips, enabled: true
+end
 
 dependency "preparation"
-dependency "chef"
-dependency "pry"
+dependency "rb-readline" if windows?
```

Comment:
I want to say there was some issue with the native gem for rb-readline on the wacky platforms, but it's worth a shot. I could be very wrong.